libs/common/src/schemas/portal-amendment.ts

```diff
@@ -0,0 +1,28 @@
+import z from 'zod';
+import { CURRENCY } from '../constants';
+import { Currency } from '../types';
+import { UNIX_TIMESTAMP_SECONDS_SCHEMA } from './unix-timestamp.schema';
+
+/**
+ * Portal Amendment schema to validate an object contains only user provided values on `PortalFacilityAmendment`
+ *
+ * If this schema is changed the Open Api docs should be updated too
+ */
+export const PORTAL_FACILITY_AMENDMENT = z
+  .object({
+    changeCoverEndDate: z.boolean(),
```

Comment:
Should this one be optional too? (`changeFacilityValue` is)

Reply:
Hmmmm, given this is all put into a `.partial` I'm not sure If any of this needs to be optional or not - will make it so for now

Reply:
Hmm yeah probably doesn't matter either way then. I guess may as well make consistent👍